service/history/api/describeworkflow/api.go

```diff
@@ -155,6 +155,15 @@ func Invoke(
 			FirstRunId:                   executionInfo.FirstExecutionRunId,
 			VersioningInfo:               executionInfo.VersioningInfo,
 		},
+		WorkflowExtendedInfo: &workflowpb.WorkflowExecutionExtendedInfo{
+			ExecutionExpirationTime: executionInfo.WorkflowExecutionExpirationTime,
+			RunExpirationTime:       executionInfo.WorkflowRunExpirationTime,
+			CancelRequested:         executionInfo.CancelRequested,
+		},
+	}
+
+	if executionInfo.OriginalExecutionRunId != executionState.RunId {
+		result.WorkflowExtendedInfo.LastResetTime = executionState.StartTime
```

Comment:
What about the `original_start_time` field?

Reply:
We didn't add it to WorkflowExtendedInfo. Which I now think was a mistake - I think StartTime ->is<- `original_start_time`, because it was coming from the event. But apparently it is coming from the new event.
So if we want `original_start_time`, we should add it to API proto. Chad will be happy I guess.

Reply:
hmm... I can see this field in api main: https://github.com/temporalio/api/blob/master/temporal/api/workflow/v1/message.proto#L131

Reply:
oh. lol I forgot that I add it after all.
Ok, added.